### PR TITLE
Converted hard-coded search results sort option "by relevance" to configuration individual.

### PR DESCRIPTION
**Pull request by litvinovg**
[Related VIVO PR](https://github.com/vivo-project/VIVO/pull/3928)
# What does this pull request do?
Converted hard-coded search results sort option "by relevance" to configuration individual.
Without that there is no way to change order of sort options on search results page.
Apply first sort option if no query text was provided and no sort option was explicitly set.

# Interested parties
@VIVO-project/vivo-committers

### CODE REVIEW

**Review by chenejac — CHANGES_REQUESTED** [review]
@litvinovg thanks for this. Couple of comments:

- when I am entering search page (from the home page, type some text and hit the button Search), the list of results is shown, it is ordered by relevance, but in the dropdown list it is "Sort by" (not "Sort by Relevance")
- should we make labels for other languages ? Meaning to create search_individuals_vitro.n3 and vitro_extended_search.ttl for other languages (with google translations for the beginning)
-  also, we need a strategy when new set labels are grouped in a new file (vitro_extended_search.ttl), and when it is just added in vitro_UiLabel.ttl

**Comment by litvinovg** [comment]
> @litvinovg thanks for this. Couple of comments:
> 
>     * when I am entering search page (from the home page, type some text and hit the button Search), the list of results is shown, it is ordered by relevance, but in the dropdown list it is "Sort by" (not "Sort by Relevance")
Yes. "Sort by" is a label for the field, not a selected value. Sorting option that was used doesn't present as chosen in the list of sorting options. Until sorting option explicitly selected sorting type is decided by contents of search query: if any text provided, then sort by relevance; if no text was provided, then search by first sort option defined by the used. If sort option wasn't set explicitly it is not saved for the user to avoid situation when at first user selected some filters, got results sorted by first defined sorting option and sorting option is saved and then added some text. In that case sorting option saved on previous step would prevent user from getting results based on scores calculated from text query.
>     * should we make labels for other languages ? Meaning to create search_individuals_vitro.n3 and vitro_extended_search.ttl for other languages (with google translations for the beginning)
Yes.

**Comment by litvinovg** [comment]
@chenejac Added translations for different languages. We need reviews from native speakers.

File: home/src/main/resources/rdf/i18n/pt_BR/display/firsttime/search_individuals_vitro.n3 [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@prefix : <https://vivoweb.org/ontology/vitro-search-individual/> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+
+:filter_group_search_filters rdfs:label "Filtros"@pt-BR .
+:filter_type rdfs:label "Tipo"@pt-BR .
+:filter_category rdfs:label "Categoria"@pt-BR .
+:sort_title_desc rdfs:label "Título Z-A"@pt-BR .
+:sort_title_asc rdfs:label "Título A-Z"@pt-BR .
+:sort_by_relevance rdfs:label "Relevância"@pt-BR .
+:field_label_sort rdfs:label "Campo de classificação de rótulo"@pt-BR .
```
Comment (salmjunior) [inline home/src/main/resources/rdf/i18n/pt_BR/display/firsttime/search_individuals_vitro.n3:10]: "Campo de ordenação por rótulo"

**Review by bkampe — CHANGES_REQUESTED** [review]

File: home/src/main/resources/rdf/i18n/de_DE/display/firsttime/search_individuals_vitro.n3 [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+
+:filter_group_search_filters rdfs:label "Suchfilter"@de-DE .
+:filter_type rdfs:label "Typ"@de-DE .
+:filter_category rdfs:label "Kategorie"@de-DE .
+:sort_title_desc rdfs:label "Titel Z-A"@de-DE .
+:sort_title_asc rdfs:label "Titel A-Z"@de-DE .
+:sort_by_relevance rdfs:label "Relevanz"@de-DE .
+:field_label_sort rdfs:label "Etikettensortierfeld"@de-DE .
+:field_category rdfs:label "KlasseGruppe"@de-DE .
```
[the remark this answers is not in the document]
Comment (litvinovg) [reply home/src/main/resources/rdf/i18n/de_DE/display/firsttime/search_individuals_vitro.n3]: Thank you. Fixed.

File: home/src/main/resources/rdf/i18n/pt_BR/display/firsttime/search_individuals_vitro.n3 (hunk 1 above)
Comment (litvinovg) [reply home/src/main/resources/rdf/i18n/pt_BR/display/firsttime/search_individuals_vitro.n3]: Thank you. Fixed translation.

**Comment by litvinovg** [comment]
I tested sorting after the discussion at the meeting. It seems to be working.

**Pull request merged**